sdk/securitycenter/Azure.ResourceManager.SecurityCenter/tests/Scenario/ConnectorSettingTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.SecurityCenter.Models;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.SecurityCenter.Tests
+{
+    internal class ConnectorSettingTests : SecurityCenterManagementTestBase
+    {
+        private ConnectorSettingCollection _connectorSettingCollection => DefaultSubscription.GetConnectorSettings();
+
+        public ConnectorSettingTests(bool isAsync) : base(isAsync, RecordedTestMode.Record)
+        {
+        }
+
+        [SetUp]
+        public void TestSetUp()
+        {
+        }
+        /// <summary>
+        /// </summary>
+        /// <returns></returns>
+        [RecordedTest]
+        [Ignore("System.InvalidOperationException : The requested operation requires an element of type 'Object', but the target element has type 'Array'.")]
```

Comment:
Is this also swagger or service issue? Can you create a github issue if yes.

Reply:
It looks like the [github issue](https://github.com/Azure/azure-rest-api-specs/issues/21260) is already created, please link the issue in the ignore description.